src/App.jsx

```diff
@@ -1,9 +1,49 @@
 import './App.css'
+import Button from "./components/btn.jsx";
+import Item from "./components/item.jsx";
+import Tile from "./components/tile.jsx";
+import {lorem} from "./assets/texts/text.js";
+
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  )
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+            <nav>
+                <Button name={"To the collection"} />
```

Comment:
Bij het meegeven van de waarde aan de properties gebruikt je óf " " (voor tekst) óf { } (voor booleans, afbeeldingen en variabelen). Dus als je " "  gebruikt hoeven er geen snorretjes { } omheen.

```suggestion
                <Button name="To the collection" />
                <Button name="Shop all bags" />
                <Button name="Pre-orders" disabled={true}/>
```